apps/search/next.config.js

```diff
@@ -15,6 +15,9 @@ const isSentryPropertiesInEnvironment =
  */
 const moduleExports = {
   reactStrictMode: true,
+  experimental: {
+    instrumentationHook: true
```

Comment:
this allows to run a script on next.js startup. In this POC only used for development to conditionally inject worker startup